src/PaymentPart/Output/MarkupOutput/Template/Html/PaymentPartTemplate.php

```diff
@@ -94,10 +93,8 @@ class PaymentPartTemplate
 /* Receipt */
 #qr-bill-receipt {
     box-sizing: border-box;
-    width: 62mm;
-	border-right: 0.2mm solid black;
-	padding-left: 5mm;
-	padding-top: 5mm;
+    width: 52mm;
+	padding: 5mm;
```

Comment:
I think this pr should not change the contents of html templates. Is it required? It seems impossible to understand why this code change was done.

Reply:
Ok, I'll try to explain my choices piece by piece.

### padding
Looking at the spec (https://www.paymentstandards.ch/dam/downloads/style-guide-en.pdf page 6) it seems to me that the receipt part should have a general padding of 5mm. 
![image](https://user-images.githubusercontent.com/2971625/90365323-e0705b80-e065-11ea-8c90-62bccfe581ab.png)

Especially for the placeholder this additional right padding right makes sense, since the bottom placeholder in the receipt part is `float: right` and like this automatically is positioned correctly.
![image](https://user-images.githubusercontent.com/2971625/90365331-e49c7900-e065-11ea-901c-5d5faeb6cb79.png)

Then we have the padding-bottom that is added with this change (also in accordance to the specs). You never know if the text gets too long and might be touching the amount part.

### width 
According to the specs the total width of the receipt part should be 62mm (52mm + 5mm padding-left + 5mm padding right). So by adding the padding property I had to shorten the width to 52mm.
I am frankly not sure about this but to my understanding the previous properties where wrong (`width: 62mm` + `padding-left: 5mm` = 67mm).
I know that there is some kind of border-box vs. content-box issue in modern browsers (https://stackoverflow.com/questions/4698054/css-does-the-width-include-the-padding) that is discussed often. To my understanding the majority of browsers adds the padding to the width  resulting in a receipt part that is 5mm too wide here. 

### border
I removed the border since I had to change the behaviour of the separators to appear when needed instead of disappear when not needed.
This also has to do with the nature of xml that does not have stylesheets. Its much harder to overwrite a previously set property without something like css. So I'd rather not set a border by default and then add a property using your `{{ printable-content }}` tag. By doing that I unfortunately had to change your html payment part template too.

Reply:
Border-box/content-box is not an "issue", it can be defined by the `box-sizing` property, as on line 96/95, which basically says «inset the padding instead of adding it to the width».